canvas_modules/common-canvas/__tests__/common-properties/operators/equalsOp-test.js

```diff
@@ -69,7 +69,7 @@ describe("validating equals operator works correctly", () => {
 		expect(equals(wrap(null), undefinedPlaceholder, null, controller)).to.equal(true);
 		expect(equals(wrap({ temp: "value" }), undefinedPlaceholder, { temp: "value2" }, controller)).to.equal(false);
 		// pass in a function as a way to hit the default switch case
-		expect(equals(wrap(emptyFunc), undefinedPlaceholder, null, controller)).to.equal(true);
+		expect(equals(wrap(emptyFunc), undefinedPlaceholder, "value", controller)).to.equal(true);
```

Comment:
Extra logic was added to operation to handle nulls.  This is needed to hit the default condition.